modules/minimp3/audio_stream_mp3.h

```diff
@@ -59,6 +59,7 @@ class AudioStreamPlaybackMP3 : public AudioStreamPlaybackResampled {
 protected:
 	virtual int _mix_internal(AudioFrame *p_buffer, int p_frames) override;
 	virtual float get_stream_sampling_rate() override;
+	_FORCE_INLINE_ int get_frames_per_beat() const;
```

Comment:
```suggestion
	int get_frames_per_beat() const;
```
AFAIK you can't force inline a method not defined in the header